### ISSUE TITLE
all players besides host fall through floor when entering scpdungeon

### ISSUE THREAD

**Issue by GodOfMelons** [Badhamknibbs/SCPCB_DunGen_LC#13]
when the scp dungeon spawns, the host can enter and see everything fine, but anyone else enters and all floors walls etc are invisible and un-interactable causing players to fall through the floor and die. configs are the same across installs

**Comment by Aleksandermyr** [comment]
Exact same issue here! I also confirmed that configs were the same on host and player pc by manually looking through them on screenshare.
I also tried uninstalling scoopysvarietymod to see if it created any difference but it didnt.
as of now i have no idea what could be causing this issue.

**Comment by Badhamknibbs** [comment]
There is a known comparability issue with Lethal Expansion that causes this, you have to either downgrade to 1.3.6 of LE (which still has bugs but is mostly functional) or not use LE.

**Comment by perfect-deform** [comment]
Nope. Same on LE 1.3.6. But on LE 1.3.6 I can clearly see that all clients have vanilla factory generated, and the host has SCP Dungeon. And ofcourse, all clients fall to death.
Both the host and clients have the same config file, same versions of mods.
```
FoundationRarity = 100
FoundationMoons = NULL
FoundationMoonsList = secret labs
FoundationGuaranteed = true
FoundationLengthOverride = -1
```

SCPCBDunGen v1.4.1
LE 1.3.6
Secret Labs 3.2.2

Not only Secret Labs has this bug. Any map do.

![photo_2024-01-04_04-35-39](https://github.com/Badhamknibbs/SCPCB_DunGen_LC/assets/60361408/c2ba9173-dbfa-44bb-b8cb-bed7556c8c62)
![photo_2024-01-04_04-35-49](https://github.com/Badhamknibbs/SCPCB_DunGen_LC/assets/60361408/8b836087-d7f4-48e4-b5ff-7219235a6fb6)

**Comment by Badhamknibbs** [comment]
Weird, testing on my end generally worked fine with 1.3.6; that said compatibility with Lethal Expansion isn't generally a consideration right now as it's simply too hard to work around how much it changes things. Some upcoming LethalLib changes might improve things but no promises with LE.

**Comment by darkmedic043** [comment]
For me it seems to only do this on modded moons,
